outputs.md

```diff
@@ -17,64 +17,101 @@ FHD outputs various data products. We outline and describe these below. <br />
+
+  * **cal.tile_a/cal.tile_b**: array of the tile index of the visibility, where tile A and tile B refer to the two separate tiles used in creating that visibility. This ordering is the same for cal.uu and cal.vv, and indices from cal.bin_offset can be used.
+
+  * **cal.gain/cal.gain_residual**: a pointer array of dimension N<sub>pol</sub> which points to an complex array of dimension N<sub>freq</sub> by N<sub>tile</sub>. These arrays contain the complex gain or residual complex gains after all fits have been applied.
+
+  * **cal.auto_params**: !Q
+
+  * **cal.convergence**: a pointer array of dimension Npol which points to an float array of dimension Nfreq by Ntile. These arrays contain a convergence measure for each pol/frequency/tile.
+
+  * **cal.amp_params/cal.phase_params/cal.mode_params**: a pointer array of dimension N<sub>pol</sub> by N<sub>tile</sub> which points to an array of coefficients used in the polynomial or mode fitting. The amplitude and phase parameters are ordered by the least-degree to the highest-degree; a polynomial fit of A + Bx + Cx<sup>2</sup> has an array of three values A,B, and C. The mode parameters are ordered by mode in index units of the Fourier transform (can be a non-integer for the hyperfine DFT option), amplitude of the mode, and phase of the mode. The wave can be reconstructed by amp * e<sup>-i2pi*(mode*[0,1,2...n_freq-1]/n_freq)+i*phase</sup>. If mode parameters (or amplitude or phase parameters) are not calculated for that tile, there will be a Null pointer in that place.
+
+  * **cal.skymodel**: a structure of the various skymodel parameters used in calibration.
+    * **include_calibration**: !Q
+    * **n_sources**: number of sources included in the calibration skymodel
+    * **catalog_name**: name of the catalog used in the calibration
+    * **galaxy_model**: integer 1/0 value of whether or not a galaxy model was used
+    * **galaxy_spectral_index**: the spectral index of the galaxy model used or NaN if no galaxy model was used
+    * **diffuse_model**: name of the diffuse model used in the calibration
+    * **diffuse_spectral_index**: the spectral index of the diffuse model used of NaN if no diffuse model was used.
+
+    * **cal.skymodel.source_list**: an array of structures of dimension N<sub>sources</sub>.
+      * **x**: !Q
+      * **y**: !Q
+      * **ra**: the RA coordinate value of the source
+      * **dec**: the DEC coordinate value of the source
+      * **ston**: not used in the calibration source list (!Q right?)
+      * **freq**: frequency of the source model
+      * **alpha**: spectral index of the source
+      * **gain**: not used in the calibration source list (!Q right?)
+      * **flag**: type codes where 0 is no flag, 1 is low confidence, and 2 is sidelobe contamination
+      * **extend**: !Q
```

Comment:
A pointer. For extended sources, the pointer references a new **source_list** structure containing all of the extended components, in the same format. For point sources, it is a Null pointer.